src/console/consoleConnection.ts

```diff
@@ -103,14 +104,21 @@ export class ConsoleConnection extends EventEmitter implements Connection {
    * @param timeout Optional. The timeout in milliseconds when attempting to connect
    *                to the Wii or relay.
    */
-  public connect(ip: string, port: number, isRealtime = false, timeout = DEFAULT_CONNECTION_TIMEOUT_MS): void {
+  public async connect(
```

Comment:
Making this async to allow us to dynamic import. I think this makes sense too since calling `connect()` still requires time and isn't immediate. I also don't think changing this to be async has any adverse affects.